File: OpenFaaS/node-function/node-function/handler.js
'use strict'

module.exports = async (event, context) => {
	let data = JSON.parse(Object.keys(event.body).pop());

	const result = {
		"data": data,
		"n. elements": Object.keys(data).length,
		"bytes": Buffer.byteLength(JSON.stringify(data))
	}

	return context
		.status(200)
		.succeed(result)
}

File: OpenFaaS/python-function/python-function/handler.py
import json

def handle(event, context):

    data = json.loads(event.body.decode())
    file_bytes = bytes(json.dumps(data), "utf-8")

    return {
        "statusCode": 200,
        "body": {
            "data": data,
            "n. elements": len(data),
            "bytes": len(file_bytes)
        }
    }
